The bcfvalidation test fails on uniqueness3.bcf out of strange reasons.
Upstream author says that there was probably a failure in the released 
files.
https://github.com/plk/biber/issues/143
Disable this file in the test.
---
 t/bcfvalidation.t |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- biber.orig/t/bcfvalidation.t
+++ biber/t/bcfvalidation.t
@@ -4,7 +4,7 @@
 use utf8;
 no warnings 'utf8';
 
-use Test::More tests => 45;
+use Test::More tests => 44;
 use XML::LibXML;
 use Biber;
 chdir('t');
@@ -15,6 +15,7 @@
 my $CFxmlschema = XML::LibXML::RelaxNG->new(location => '../data/schemata/bcf.rng');
 
 foreach my $bcf (<tdata/*.bcf>) {
+  next if ($bcf eq "tdata/uniqueness3.bcf");
 # Set up XML parser
   my $CFxmlparser = XML::LibXML->new();
 
